src/site/markdown/server/manage-cli.md

```diff
@@ -0,0 +1,264 @@
+Manage James via the Command Line
+=================================
+
+With any distribution, James is packed with a command line client.
+
+To use it enter, for Spring distrubution:
+
+```
+./bin/james-cli.sh -h 127.0.0.1 -p 9999 COMMAND
+```
+
+And for Guice distributions:
+
+```
+java -jar /root/james-cli.jar -h 127.0.0.1 -p 9999 COMMAND
+```
+
+The following document will explain you which are the available options for **COMMAND**.
+
+Note: the command line before **COMMAND** will be documente as *{cli}*.
+
+## Manage Domains
+
+Domains represent the domain names handled by your server.
+
+You can add a domain:
+
+```
+{cli} AddDomain domain.tld
+```
+
+You can remove a domain:
+
+```
+{cli} RemoveDomain domain.tld
+```
+
+Check if a domain is handled:
+
+```
+{cli} ContainsDomain domain.tld
+```
+
+And list your domains:
+
+```
+{cli} ListDomains
+```
+
+## Managing users
+
+Users are accounts on the mail server. James can maintain mailboxes for these mailaddress.
+
+You can add a user:
+
+```
+{cli} AddUser user@domain.tld password
+```
+
+Note: the domain used should have been previously created.
+
+You can delete a user:
+
+```
+{cli} RemoveUser user@domain.tld
+```
+
+And reset its password:
+
+```
+{cli} SetPassword user@domain.tld password
+```
+
+Note: All these write operations can not be performed on LDAP backend, as the implementation is read-only.
+
+Finally, you can list users:
+
+```
+{cli} ListUsers
+```
+
+## Managing mailboxes
+
+An administrator can perform some basic operation on user mailboxes.
+
+Note on mailbox formatting: mailboxes are composed of three parts.
+
+ - The namespace, indicating what kind of mailbox it is. (Shared or not?). The value for users mailboxes is #private
```

Comment:
I don't know, we don't support shared mailboxes yet ;-) 

(Thus it is pointless to specify it)